src/Telegram/Types/ChatMember/ChatMemberBanned.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace unreal4u\TelegramAPI\Telegram\Types\ChatMember;
+
+use unreal4u\TelegramAPI\Telegram\Types\ChatMember;
+
+/**
+ * Represents a chat member that was banned in the chat and can't return to the chat or view chat messages.
+ *
+ * @see https://core.telegram.org/bots/api#chatmemberbanned
+ */
+class ChatMemberBanned extends ChatMember
+{
+    public const STATUS = 'kicked';
```

Comment:
In the same vain of keeping this as close to the source as possible, it would be better not to define constants here (or in any of the other `ChatMember` classes).

A simpler and more consistent way would be to define the status in the same way we do here: 
https://github.com/unreal4u/telegram-api/blob/553e2c53d346454b96f27fbf67f080d7efbae13a/src/Telegram/Types/InputMedia/Video.php#L22 by just assigning it directly to the public property.